packages/@sanity/components/src/panes/SplitController.js

```diff
@@ -8,7 +8,7 @@ const COLLAPSED_WIDTH = 54
 
 export default class PanesSplitController extends React.Component {
   static propTypes = {
-    children: PropTypes.node.isRequired,
```

Comment:
I think the propType warning comes from 'Pane' and not the 'SplitController'

Reply:
Yep, added it back